ote/src/clj/ote/transit_changes/detection.clj

```diff
@@ -695,8 +726,8 @@
         removed? (and last-route-change? (max-date-within-evaluation-window? route))
         no-traffic? (and no-traffic-start-date
```

Comment:
could no-traffic? be renamed to break-in-traffic?

Reply:
Will consider in next PR which is a-brewing